examples/forms.amp.html

```diff
@@ -270,12 +270,16 @@
       </template>
     </div>
     <div submit-success>
-      Success! Thanks for subscribing! Please make sure to check your email
```

Comment:
This was changed in #8565 for a feature request #5619, so this is supposed to be a supported. I think I failed to update the validator rules for it. We should change the validator protoascii to support this use case